v8go.cc

```diff
@@ -198,6 +198,25 @@ IsolateHStatistics IsolationGetHeapStatistics(IsolatePtr iso) {
                             hs.number_of_detached_contexts()};
 }
 
+/********** Exceptions & Errors **********/
+
+ValuePtr ThrowException(ValuePtr value) {
+  Isolate* iso = value->iso;
+  m_ctx* ctx = value->ctx;
```

Comment:
One convention that we may want to follow is that the function in v8go.cc is usually named with the object it belongs to as a prefix and as it's first argument, like:
```cpp
void IsolateDispose(IsolatePtr iso) {
  if (iso == nullptr) {
    return;
  }
  ContextFree(isolateInternalContext(iso));
  iso->Dispose();
}
```
i.e.
```cpp
ValuePtr IsolateThrowException(IsolatePtr iso, ValuePtr value) {
  if (iso == nullptr) {
    return;
  }
...
}
```
then in the Go function:
```go
func (i *Isolate) ThrowException(value *Value) *Value {
	return &Value{
		ptr: C.ThrowException(i.ptr, value.ptr),
	}
}
```
This way we don't rely on the value's field to get an isolate and risk a nil pointer error because the isolate in go is a pointer that the function `ThrowException` would be called on and it would fail at the go level if the isolate is nil instead of within the c++ code. (There are probably other reasons that are slipping my mind right now.)